src/ResourceManager/Sql/Commands.Sql.Test/ScenarioTests/DatabaseBackupTests.ps1

```diff
@@ -276,4 +276,82 @@ function Test-DatabaseGeoBackupPolicy
 	Set-AzureRmSqlDatabaseGeoBackupPolicy -ServerName $server.ServerName -ResourceGroupName $rg.ResourceGroupName -DatabaseName $db.DatabaseName -State Disabled
 	$result = Get-AzureRmSqlDatabaseGeoBackupPolicy -ServerName $server.ServerName -ResourceGroupName $rg.ResourceGroupName -DatabaseName $db.DatabaseName
 	Assert-True { $result.State -eq "Disabled" }
+}
+
+function Test-NewDatabaseRestorePoint
+{
+	# Setup
+	$location = "West US 2"
```

Comment:
Should use Get-Location: https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Common/Commands.ScenarioTests.ResourceManager.Common/Common.ps1#L583. Please be sure to include all three parameters.

Reply:
done